dataverse-persistence/src/main/resources/db/migration/V86__add_geographic_coordinates.sql

```diff
@@ -0,0 +1,59 @@
+create table gebox_coordinates_tmp as (
+select d.*, 'westLongitude' as fieldname  from datasetfield d join datasetfieldtype dt on d.datasetfieldtype_id = dt.id where dt.name = 'westLongitude'
+union
+select d.*, 'eastLongitude' as fieldname from datasetfield d join datasetfieldtype dt on d.datasetfieldtype_id = dt.id where dt.name = 'eastLongitude'
+union
+select d.*, 'northLongitude' as fieldname from datasetfield d join datasetfieldtype dt on d.datasetfieldtype_id = dt.id where dt.name = 'northLongitude'
+union
+select d.*, 'southLongitude' as fieldname from datasetfield d join datasetfieldtype dt on d.datasetfieldtype_id = dt.id where dt.name = 'southLongitude'
+);
+
+
+INSERT INTO public.datasetfieldtype
+(advancedsearchfieldtype, allowcontrolledvocabulary, allowmultiples, description, displayformat, displayoncreate, displayorder, facetable, fieldtype, "name", required, title, uri, watermark, metadatablock_id, parentdatasetfieldtype_id, inputrenderertype, inputrendereroptions, validation, metadata, exporttofile, visiblethroughanonymizedurl)
+VALUES(true, false, false, 'Enter geographic coordinates to mark dataset location', '', true, 15, false, 'TEXTBOX', 'geographicCoordinates', true, 'Geographic coordinates', NULL, '', 2, 85, 'TEXTBOX', '{}', '[{"name":"geobox_polygon_component_validator", "parameters":{"runOnEmpty":"true"}}]', '{}', false, true);
```

Comment:
Do we really want for this field to be `required` and `displayoncreate` for all of the instances?